neo/Network/P2P/Capabilities/NodeCapabilityBase.cs

```diff
@@ -22,14 +22,19 @@ protected NodeCapabilityBase(NodeCapabilities type)
             Type = type;
         }
 
-        public virtual void Deserialize(BinaryReader reader)
+        public virtual void Deserialize(BinaryReader reader) { }
```

Comment:
This should include `Type`, otherwise `Neo.IO.Helper.AsSerializable<T>()` may fail.

https://github.com/neo-project/neo/blob/8b49c507334df9ff64fa99fbde02c30eda91469e/neo/IO/Helper.cs#L15-L22